src/AudioAnalysisTools/UnitConverters.cs

```diff
@@ -127,7 +127,7 @@ public UnitConverters(double segmentStartOffset, int sampleRate, int frameSize,
         /// <remarks>
         /// Measured in seconds per spectrogram frame.
         /// </remarks>
-        public LinearSecondsScale SecondsScale { get; }
+        //public LinearSecondsScale SecondsScale { get; }
```

Comment:
remove this line, it is not used, and further is related to a flawed understanding

Reply:
THis line and comment have been removed.